contracts/merkleToken.scrypt

```diff
@@ -0,0 +1,133 @@
+import "util.scrypt";
+import "./merkleTree.scrypt";
+
+contract merkleToken {
+  int price;
+
+  public function buyMore(SigHashPreimage txPreimage,
```

Comment:
Please add some comments on the difference btw `buyMore` and `buy`. Also `sell`.